### PR TITLE
requirements.txt

### LINKED ISSUE #60: Install project  dependencies
This project has dependencies it would be good to add those into a `requirement.txt` file and update Readme to install same to setup development environment

### PR DESCRIPTION
**Pull request by vatsa287**
Fixes: #60

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+requests
+xxhash
+grpcio
+pyxattr
+kubernetes
+#any feedbacks on version_no?
```
Comment (amarts) [inline requirements.txt]: No need to have this here.. it should be a comment while reviewing. Remove this.

**Review by amarts — COMMENTED** [review]
Also, commit msg can have 'Fixes: #60' instead of 'Issue #60'.

File: requirements.txt (hunk 1 above)
Comment (vatsa287) [reply requirements.txt]: Updated

**Review by amarts — COMMENTED** [review]
/lgtm

**Review by aravindavk — APPROVED** [review]

**Pull request merged**